CMakeLists.txt

```diff
@@ -1210,6 +1212,7 @@ function(add_library_variant target_arch)
             message("C++ runtime libraries tests disabled for ${variant}")
         else()
             add_custom_target(check-llvm-toolchain-runtimes-${variant})
+            add_dependencies(check-llvm-toolchain check-llvm-toolchain-runtimes-${variant})
```

Comment:
Wouldn't it be simpler to just make `check-llvm-toolchain` depend on `check-llvm-toolchain-runtimes`, than to rely on each variant individually?